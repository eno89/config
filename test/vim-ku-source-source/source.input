" Test: vim-ku-source-source source
runtime! plugin/ku.vim plugin/ku/source/source.vim
call vspec#hint({'scope': 'ku#_local_variables()', 'sid': 'ku#_sid_prefix()'})
function s:describe__ku_source_source_gather_candidates()  "{{{1
  It should return the names of available sources regardless of args

  Should ku#source#source#gather_candidates({'pattern': 'litil'})
  \ ==# map(copy(ku#available_source_names()), '{"word": v:val}')
endfunction




" __END__  "{{{1
" vim: filetype=vim foldmethod=marker
