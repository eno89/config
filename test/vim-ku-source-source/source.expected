==== ku_source_source_gather_candidates 
---- It should return the names of available sources regardless of args 
.



**** Result ****


1 examples, 0 failures
